src/app/shared/service/reservation.service.ts

```diff
@@ -32,4 +30,27 @@ export class ReservationService {
         return [];
       });
   }
+  formatDate(date: Date) {
+    const year = date.getUTCFullYear();
+    const month = String(date.getUTCMonth() + 1).padStart(2, '0');
+    const day = String(date.getUTCDate()).padStart(2, '0');
+    const hours = String(date.getHours()).padStart(2, '0');
+    const minutes = String(date.getUTCMinutes()).padStart(2, '0');
+    const seconds = String(date.getUTCSeconds()).padStart(2, '0');
+
+    return `${year}-${month}-${day}T${hours}:${minutes}:${seconds}`;
+  }
+
+  makeReservation(reservation:{
+    wingId: string,
+    startDateTime: Date,
+    endDateTime: Date, }){
+    return this.apiService.post<ApiResponse<any>>('/reservations/reserve-workplace', {
```

Comment:
Het is beter als je ook nog een model meegeeft aan de ApiResponse<>

Reply:
kan niet, want op het moment van reserveren heb ik nog geen location. en als k die meegeef dan gebruik ik de model alleen voor start en eind datum